### PR TITLE
Add basic template to repo

### PR DESCRIPTION
**Pull request by leslietilling**
## Purpose of this pull request

Our wiki was linking to template in DevDocs, which does not match many of our conventions and resource links.

For new images, I added an `assets` folder to the getting-started directory. This is the model that the Adobe repos are using and it also a better organizational structure than one big massive images folder. We will be gradually moving to this model. 

If Magento Commerce, please specify:

- [ ] Commerce only?
- [ ] Commerce with B2B?

## Affected documentation pages

N/A

### CODE REVIEW

File: .github/TOPIC_TEMPLATE [hunk 1]
```diff
@@ -0,0 +1,222 @@
+---
+title: Basic template
+---
+
+Introductory text that gives an overview of the topic you will be writing about.
+
+The purpose of this page is to provide you with a pre-formatted template and useful [markdown](https://glossary.magento.com/markdown) references to help you get started writing docs.
+
+You can start off by editing the local version of this file using markdown language (and [HTML](https://glossary.magento.com/html) where needed). Then, create a Pull Request to have your contribution reviewed by the Docs team.
```
Comment (hickskenh) [inline .github/TOPIC_TEMPLATE:9]:
```suggestion
You can start by editing the local version of this file using markdown language (and [HTML](https://glossary.magento.com/html) where needed). Then, create a pull request (PR) to have your contribution reviewed by the Docs team.
```

File: .github/TOPIC_TEMPLATE [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+
+Output:
+
+1. First Step
+1. Second Step
+1. Third Step
+
+### Images {#images}
+
+Please add any images you may need to the [`src/images`](https://github.com/magento/merchdocs/tree/master/src/images) directory.
```
Comment (hickskenh) [inline .github/TOPIC_TEMPLATE:80]:
```suggestion
Add any images you may need to the [`src/images`](https://github.com/magento/merchdocs/tree/master/src/images) directory.
```

also, don't some projects use the Assets folder?

Comment (leslietilling) [reply .github/TOPIC_TEMPLATE:80]: For now that is the primary place. We can adjust location for now if there is an assets folder.

**Comment by leslietilling** [comment]
running tests

**Pull request merged**